docker/deployment/Dockerfile

```diff
@@ -1,15 +1,8 @@
 FROM ubuntu:22.04
 
 RUN apt-get update && \
-    apt-get install -y curl jq nodejs npm python3-pip
+    apt-get install -y curl jq
 
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg && \
     apt-get update && apt-get install -y google-cloud-sdk
-
-RUN pip3 install pipenv==2023.12.1
-# Install the latest npm version 8 release (last release is in Feb 2023)
-# This will take precedence over the system installed version on the PATH
-RUN npm install -g npm@8
-
-COPY --from=gcr.io/oss-vdb/ci /root/go/bin/hugo /usr/local/bin/hugo
```

Comment:
is it intentional not to `npm` or copy `hugo`?

Reply:
Yes, these dependencies were originally included because we deployed the website through appengine using this deployment image. This is no longer necessary, as we build everything in appengine's own image now, and only use the deployment image to upload to cloudrun.